controllers/compassmanager_controller_test.go

```diff
@@ -27,6 +27,27 @@ var _ = Describe("Compass Manager controller", func() {
 	kymaCustomResourceLabels := make(map[string]string)
 	kymaCustomResourceLabels["operator.kyma-project.io/managed-by"] = "lifecycle-manager"
 
+	It("handles `compass-runtime-id-for-migration`", func() {
```

Comment:
When creating a test case we follow this pattern: Describe -> Context -> It / DescribeTable. Could you adjust this test case to the pattern? You can check how the other tests are described.